styles.css

```diff
@@ -0,0 +1,5 @@
+
+.navbar{
+    background-color: #be3144;
+
+}
```

Comment:
No new line at the end of the file; it would be more professional to add that line.